### DISCUSSION TITLE
Council size

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#16] [Q&A]
Minimum 3, max 1_000_000? 
It becomes unmanageable after a certain size, I'd recommend a size limit.

**Comment by ebullient** [comment]
Sure. I will let the bootstrapping group determine that cap. Sounds like you want to be on it. ;)

Apache has 9, Gnome Foundation has 7. Eclipse Foundation Board size varies, as every "strategic member" has a seat on the board.

I did have a recommendation to define a floor, and leave it open to the bootstrapping council (of that minimum size) to define a ceiling. I personally would shoot for 7-11, to  make it easier to reach quorum and spread the load.

**Comment by kenfinnigan** [comment]
As mentioned by @ebullient, a floor is currently set [here](https://github.com/commonhaus/foundation-draft/blob/main/bylaws/3-cf-council.md#composition-and-membership). Setting a ceiling can be done through by-law amendment once the foundation is running during the bootstrap phase
